rules/private/phases/phase_javainfo.bzl

```diff
@@ -28,15 +28,6 @@ def phase_javainfo(ctx, g):
     if len(ctx.attr.srcs) == 0 and len(ctx.attr.resources) == 0:
         java_info = java_common.merge([g.classpaths.sdeps, sexports, sruntime_deps])
     else:
-        # TODO: why do ijars break Scala 3?
-        # For some yet unknown reason ijars break Scala 3.
-        # Bazel now handles .tasty files, but the Scala 3 test fails to pass
-        # when this ijar is used as the compile jar. My guess is that the
-        # classfile format changed somehow for Scala 3 and Bazel does not yet
-        # handle that.
-        #
-        # In the meantime, we've added a use_ijar Scala configuration value and
```

Comment:
Did you consider removing the `use_ijar` configuration altogether?

Reply:
I have, but I want to get this merged and try it out in our main repo before I remove it. The rest I wrote here no longer fails, but I'll need more evidence to feel more confident.